server/plugin/plugin.go

```diff
@@ -966,12 +966,11 @@ func (p *Plugin) HasUnreads(info *GitHubUserInfo) bool {
 	return true
 }
 
-func (p *Plugin) checkOrg(org string) error {
-	config := p.getConfiguration()
-
-	configOrg := strings.TrimSpace(config.GitHubOrg)
-	if configOrg != "" && configOrg != org && strings.ToLower(configOrg) != org {
-		return errors.Errorf("only repositories in the %v organization are supported", configOrg)
+func (p *Plugin) checkOrg(org, configOrg string, orgMap map[string]bool) error {
```

Comment:
It seems odd pass the list of orgs to `checkOrg`. Can the `checkOrg` fetch the list of orgs itself?

Reply:
`checkOrg` is used within loop at various places in the code, like: https://github.com/mattermost/mattermost-plugin-github/blob/49ad56d4ceb246ca5f420a7183a41955132bc9a0/server/plugin/api.go#L678. If we include fetching of list from within the function, it will only increase the time complexity of the code.

Reply:
The config is stored in memory. Hence, accessing it fast. Is the additional code complex worth the performance improvements?